doc/classes/ProjectSettings.xml

```diff
@@ -1320,6 +1320,10 @@
 			If [code]false[/code], no input will be lost.
 			[b]Note:[/b] You should in nearly all cases prefer the [code]false[/code] setting. The legacy behavior is to enable supporting old projects that rely on the old logic, without changes to script.
 		</member>
+		<member name="input_devices/joypad/enabled" type="bool" setter="" getter="" default="true">
+			If [code]true[/code], the joypad will be auto connected as Input device.
+			If [code]false[/code], Godot will not connect to joypads.
```

Comment:
```suggestion
			If [code]true[/code], Godot will connect to joypads.
```
I'd say a more direct description, and at least it should be a single paragraph, each new line is a paragraph

Reply:
Thanks!